packages/console/src/pages/SignInExperience/components/GuideModal.tsx

```diff
@@ -28,10 +29,12 @@ import TermsForm from './TermsForm';
 
 type Props = {
   isOpen: boolean;
+  languages: LanguageTag[];
```

Comment:
what about rename to `languageTags` to correspond with the type name

Reply:
updated.